### ISSUE TITLE
Number in Collection Rarity filter broken

### ISSUE THREAD

**Issue by jeff-wichers** [TakaOtaku/Digimon-Card-App#533]
I am both unable to edit the filter value and unable to see cards for which I have more than 5 copies.

**Comment by TakaOtaku** [comment]
I should have pushed a fix for that yesterday, please try if it works.
The error is the new attribute for the Collection Maximum, which can be set in the settings. When it is not set, the cards get filtered out when above 5.
If it is still broken please comment :)

**Comment by jeff-wichers** [comment]
I did find [closed issue 531](https://github.com/TakaOtaku/Digimon-Card-App/issues/531), yesterday. At the time that I posted, I was experiencing the same issue. But I was able to resolve the issue using the steps they reported [here](https://github.com/TakaOtaku/Digimon-Card-App/issues/531#issuecomment-1983939335). The value was set to 5 for me by default. I tried changing it to 300 and no change in the behavior. But I cleared it, saved, set it to 5 again and saved, and it worked correctly after that.

**Comment by TakaOtaku** [comment]
Yeah that was mistake, I did find the workaround and then fix it. But I have released it on my test server not on the main server D:

**Comment by TakaOtaku** [comment]
Even if you delete the Collection Max now, it shouldn't destroy the collection filter
